.github/workflows/pr-labeler.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         name: Changeset status
         continue-on-error: true
         run: |
-          changeset status --since=${{ env.GITHUB_REF_NAME }}
+          changeset status --since=${{ github.base_ref }}
```

Comment:
Nit: It looks like this template is missing root-level docs. Probably worth adding (not necessarily in this PR), and probably worth documenting required variables at that level.